mkdocs.yml

```diff
@@ -110,8 +111,15 @@ extra:
       link: https://dietpi.com/
     - icon: fontawesome/brands/twitter
       link: https://twitter.com/DietPi_
+  item076:
+      name: LAMP Webstack
+      display_name: LAMP Webstack - Apache2 / MariaDB / PHP
```

Comment:
Let's make `display_name` => `short_desc: Apache2 / MariaDB / PHP`

So we can use the short one-line description separately, e.g. if the dash is not a possible separator in certain cases. I don't know a specific case but it allows more flexibility.

Reply:
it's ok, updated